### ISSUE TITLE
How about add update features

### ISSUE THREAD

**Issue by suutaku** [CleanPegasus/verkle-tree-rust#1]
Hey, I like your crate, it's very clean and simple. Do you have any plan to add  tree updating feature to you library, also maybe some utils like hash to element?

**Comment by CleanPegasus** [comment]
Yes. I plan to add tree updating features this crate right after figuring out 
- multiproof with random evaluation
- KZG commitment with pre-computed powers of tau.

I'll add this to the todo section of the README
